habits/tasks.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+from celery import shared_task
+from django.utils import timezone
+
+from .models import Habit
+from .services import send_telegram_message
+
+
+@shared_task()
+def send_reminder_habits():
+    now = timezone.now()
+    current_time = now.time()
+
+    habits = Habit.objects.filter(user__isnull=False, user__tg_chat_id__isnull=False)
```

Comment:
Достаточно одной проверки, причем для оптимизации запросов лучше добавить `select_related`
```suggestion
    habits = Habit.objects.select_related('user').filter(user__tg_chat_id__isnull=False)
```
Иначе ты при каждой итерации будешь делать запрос за пользователем (100 привычек = +100 запросов, неоптимально)